mobile/lib/features/trade/trade_theme.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+
+@immutable
+class TradeTheme extends ThemeExtension<TradeTheme> {
+  // Unfortunately shades of Colors in flutter are not constant; see https://github.com/flutter/flutter/issues/31351
+  // Workaround to make is constant: Initialize the color from HEX
+  // For color codes see: https://api.flutter.dev/flutter/material/Colors/green-constant.html
+  static const Color green600 = Color(0xFF43A047);
+  static const Color red600 = Color(0xFFE53935);
+
+  const TradeTheme({
+    this.buy = green600,
+    this.sell = red600,
+    this.tradeButtonShape = const RoundedRectangleBorder(
+      borderRadius: BorderRadius.all(
+        Radius.circular(8),
+      ),
+    ),
+    this.tradeButtonWidth = 100.0,
+  });
+
+  final Color buy;
+  final Color sell;
+  final ShapeBorder tradeButtonShape;
+  final double tradeButtonWidth;
```

Comment:
note: Adding the button's width and shape to the theme could be disputed - usually the theme is used for Colors and Fonts; see: https://docs.flutter.dev/cookbook/design/themes

My idea is to create Themes per feature for now, and that's what motivated me to add more to the theme. We can review this practice once the application grows.

Reply:
Do you expect this being used for multiple buttons? 
If only used for the buy/sell FABs I don't think this should be in here.